domain/src/main/kotlin/app/discord/user/dto/GuildMemberLeaveEvent.kt

```diff
@@ -0,0 +1,9 @@
+package app.discord.user.dto
+
+import java.time.OffsetDateTime
+
+data class GuildMemberLeaveEvent(
+    val guildId: String,
+    val userId: String,
+    val leaveTime: OffsetDateTime
+)
```

Comment:
guildId 와 userId 값을 하나로 묶어놓은 [`UserIdentifier`](https://github.com/StudyManagerBot/ManagerBot/blob/develop/domain/src/main/kotlin/app/discord/user/dto/UserIdentifier.kt) 클래스가 존재합니다.
```suggestions
data class GuildMemberLeaveEvent(
    val userIdentifier: Useridentifier,
    val leaveTime: OffsetDateTime
)
```
변경사항 고려해보시기 바랍니다.